modules/listScore.js

```diff
@@ -0,0 +1,26 @@
+import plusScore from './addScores.js';
+
+const sumScores = () => {
+  const ScoresAll = document.getElementById('points');
```

Comment:
- [ ] Please I notice this error when I run your code please fix it 

![image](https://user-images.githubusercontent.com/74744787/181822635-af0bae9b-416e-4b4f-b2b8-b0a8c0cdaf0e.png)

Reply:
i tried jest on this module and no found where is the issue, there is any hint to continue ?